core/src/test/scala/com/mesosphere/usi/core/logic/MesosEventsLogicTest.scala

```diff
@@ -14,13 +14,14 @@ class MesosEventsLogicTest extends UnitTest {
 
   private val mesosEventLogic = new MesosEventsLogic(new MesosCalls(MesosMock.mockFrameworkId))
 
-  val podWith1Cpu256Mem: PodSpec = PodSpec(
-    PodId("mock-podId"),
-    Goal.Running,
-    RunSpec(
-      List(ScalarRequirement(ResourceType.CPUS, 1), ScalarRequirement(ResourceType.MEM, 256)),
-      shellCommand = "sleep 3600")
-  )
+  def podWith1Cpu256Mem(id: String = "mock-podId"): PodSpec =
```

Comment:
here and everybody else: could we please generate some random ids by default? I know, these are not IT but this will 100% copy-pasted later. We don't need the UUID, it can be something alphanumeric too:
```suggestion
  def podWith1Cpu256Mem(id: String = s"mock-podId-${scala.util.Random.alphanumeric.take(10).mkString}"): PodSpec =
```

Reply:
@zen-dog let's bring that up as a requirement for the fixture story 

https://jira.mesosphere.com/browse/DCOS-50134

Reply:
(I added it to the requirements)